client/app/components/form/MultiStepButtons.tsx

```diff
@@ -4,19 +4,19 @@ import { Button } from "@mui/material";
 import { useSession } from "next-auth/react";
 import Link from "next/link";
 
-interface SubmitButtonProps {
+interface MultiStepButtonsProps {
   baseUrl: string;
   cancelUrl: string;
   classNames?: string;
   disabled?: boolean;
   isSubmitting: boolean;
-  step: number;
+  step: number; // 0-indexed
```

Comment:
My tests kept failing because I didn't realize this, so added some comments as a reminder

Reply:
https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=61566260

Reply:
Thanks, I think the `MultiStepFormBase` was refactored to make this easier to follow and we didn't touch this 😬